backend/model/app.py

```diff
@@ -0,0 +1,16 @@
+from flask import Flask
+from flask_restful import Api
+from flask_cors import CORS
+from flask_caching import Cache
+
+
```

Comment:
it's a good practise to create a main function as entry point `if __name__ == "__main__":`

Reply:
Entry point is given in main.py file